src/test/java/cloud/fogbow/ras/core/cloudconnector/LocalCloudConnectorTest.java

```diff
@@ -1,942 +1,1254 @@
-        Mockito.when(this.order.getInstanceId()).thenReturn(null);
-        Mockito.when(this.order.getId()).thenReturn(FAKE_ORDER_ID);
-        Mockito.when(this.order.getOrderState()).thenReturn(OrderState.PENDING);
+        Order order = createLocalPublicIpOrder(FAKE_COMPUTE_ID);
+        order.setOrderState(OrderState.FULFILLED);
+
+        CloudUser cloudUser = Mockito.mock(CloudUser.class);
+        Mockito.when(this.mapperPlugin.map(Mockito.any(SystemUser.class))).thenReturn(cloudUser);
 
         // exercise
-        OrderInstance instance = this.localCloudConnector.getInstance(order);
+        this.localCloudConnector.getInstance(order);
 
-        //verify
-        Assert.assertEquals(FAKE_ORDER_ID, instance.getId());
-        Assert.assertEquals(InstanceState.DISPATCHED, instance.getState());
-        Mockito.verify(computePlugin, times(0)).getInstance(Mockito.any(ComputeOrder.class), Mockito.any(CloudUser.class));
-        Mockito.verify(volumePlugin, times(0)).getInstance(Mockito.any(VolumeOrder.class), Mockito.any(CloudUser.class));
-        Mockito.verify(attachmentPlugin, times(0)).getInstance(Mockito.any(AttachmentOrder.class), Mockito.any(CloudUser.class));
-        Mockito.verify(networkPlugin, times(0)).getInstance(Mockito.any(NetworkOrder.class), Mockito.any(CloudUser.class));
+        // verify
+        Mockito.verify(this.localCloudConnector, Mockito.times(BaseUnitTests.RUN_ONCE))
+                .createEmptyInstance(Mockito.eq(order));
+        Mockito.verify(this.localCloudConnector, Mockito.times(BaseUnitTests.RUN_ONCE)).auditRequest(
+                Mockito.eq(Operation.GET), Mockito.eq(ResourceType.PUBLIC_IP), Mockito.any(SystemUser.class),
+                Mockito.anyString());
     }
-
-    // test case: The order doesn't have an InstanceID, so an empty AttachmentInstance is returned with the same id of order.
-    // The order state is FAILED_AFTER_SUCCESSFUL_REQUEST, so the instance state must be FAILED_AFTER_SUCCESSFUL_REQUEST.
+    
+    // test case: When invoking the deleteInstance method with a valid compute order
+    // it must call the doDeleteInstance method, and confirm in auditRequest the
+    // DELETE operation and the COMPUTE resource type.
     @Test
-    public void testGetEmptyAttachmentInstanceWithFailedOrder() throws FogbowException {
-
+    public void testDeleteInstanceWithComputeOrder() throws FogbowException {
         // set up
-        this.order = Mockito.mock(AttachmentOrder.class);
-        Mockito.when(this.order.getType()).thenReturn(ResourceType.ATTACHMENT);
-        Mockito.when(this.order.getInstanceId()).thenReturn(null);
-        Mockito.when(this.order.getId()).thenReturn(FAKE_ORDER_ID);
-        Mockito.when(this.order.getOrderState()).thenReturn(OrderState.FAILED_AFTER_SUCCESSFUL_REQUEST);
+        Order order = createLocalComputeOrder();
+        order.setInstanceId(FAKE_INSTANCE_ID);
+
+        CloudUser cloudUser = Mockito.mock(CloudUser.class);
+        Moc
```

Comment:
verify if the message is the expected one

Reply:
same thing to the other unsucessful tests

Reply:
I'll check the expected message of the methods that throw exceptions in this class, the exception messages thrown inside the plugin will not be evidenced in the tests.